dashboard/test/fixtures/script.yml

```diff
@@ -221,36 +221,69 @@ course4:
 cspunit1:
   id: 53
   name: cspunit1
-  created_at: 2015-10-16 22:38:51.000000000 Z
-  updated_at: 2015-10-16 22:38:51.000000000 Z
-  wrapup_video_id:
+  created_at: 2015-10-02 19:55:31.000000000 Z
+  updated_at: 2016-05-12 19:37:06.000000000 Z
+  wrapup_video_id: 
```

Comment:
Is there a reason for the extra whitespace? For changing single quotes to double quotes?